lib/modules.js

```diff
@@ -173,7 +173,9 @@ export async function updateSourceFiles ({
       break
     } catch (err) {
       if (gateway === gateways[gateways.length - 1]) {
-        throw err
+        throw new Error(
+          `[${module}] Can't download module sources from any gateway. Last error: ${err.stack}`
+        )
```

Comment:
Can we/should we store the original error in the "cause" property of the new error?

Reply:
good idea, we can use that now :)